### ISSUE TITLE
placeholders not working

### ISSUE THREAD

**Issue by comfysquirrel** [sgtcaze/NametagEdit#111]
### What version/build of NametagEdit are you using? Type /ver NametagEdit
`4.1.2`
### What version of Bukkit/Spigot/PaperSpigot are you using? Type /ver
`CraftBukkit version git-Spigot-f19bb2c-2b7808f (MC: 1.11) (Implementing API version 1.11-R0.1-SNAPSHOT)`
### What plugins are you using? Type /plugins
`PermissionsEx, PlugMan, NametagEdit, ServerListPlus, Vault, Essentials, EssentialsSpawn, PlaceholderAPI, EssentialsChat, EssentialsGeoIP`
### Please explain your issue?
I might just misunderstand how placeholders are supposed to work. In the nte groups config I have:
```
Groups:
  staff:
    Permission: nte.staff
    Prefix: '&4[Staff] &c'
    Suffix: ' (%player_displayname%)'
    SortPriority: 1
```
but the `%player_displayname%` is showing up literally in tab, not being transformed into the placeholder.
### Is there an error(s) in the console? Please use: https://pastebin.com

**Comment by sgtcaze** [comment]
Hmm.  Do the other plugins that hook into PlaceholderAPI correctly process the %player_displayname% placeholder? Or is it just NametagEdit?

**Comment by sgtcaze** [comment]
No update in 12 days. If the issue persists, please reopen the issue.

**Comment by comfysquirrel** [comment]
I solved it using the same solution as #125: I needed to download the placeholderapi extension for the plugin I wanted the placeholder from
